index.html

```diff
@@ -0,0 +1,250 @@
+
+                        <span>2015</span>
+                    </div>
+                    <div class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="content">
+                    <div class="topic">
+                        Multi-Post
+                        <br />
+                        Stories
+                    </div>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <div class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+                <img class="img" src="image/card-2.svg" alt="" />
+            </div>
+
+            <div class="card">
+                <img class="img" src="image/card-3.svg" alt="" />
+                <div class="content">
+                    <div class="topic">Tonic</div>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <div class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </div>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project">See Project</div>
+                </div>
+            </div>
+
+            <div class="card">
```

Comment:
- [ ] Kindly ensure that your project card layout matches the Figma design template chosen  👍🏼 . Kindly add the appropriate padding, color, font size and position the elements properly 👍🏼 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/185760602-51e02053-2a11-4702-83a7-a0f782d009d8.png)  |![image](https://user-images.githubusercontent.com/63473584/185760666-10db12e6-34a5-4d63-819c-5a53cd5773c1.png)  |

You will be able to position the elements nice and neat using flexbox which will also make the section semantic 😄